projects/media-viewer/src/lib/icp/icp-session.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class IcpSessionService {
+
+  private ICP_SESSION_API = '/icp/sessions';
+
+  constructor(private readonly httpClient: HttpClient) {}
+
+  public loadSession(caseId: string): Observable<any> {
+    return this.httpClient
+      .get(`${this.ICP_SESSION_API}/${caseId}`,
```

Comment:
it would be useful to include the return type here with the get 
httpClient.get<>